operator/docs/user-guides/forwarding_logs_to_gateway.md

```diff
@@ -134,6 +134,59 @@ _Note: While this document will only give instructions for two methods of log fo
 
     _Note:_ You can add/remove any pipeline from the ClusterLogForwarder spec in case if you want to limit the logs being sent.
 
+## Network Observability
+
+[Network Observability](https://github.com/netobserv/network-observability-operator) also require an external loki instance and is compatible with LokiStack Gateway.
+
+* Deploy the Loki Operator and `lokistack` using `openshift-network` mode and `lokistack-network` name for example. You must use a separate instance than `openshift-logging` one. 
+
+* Create the following `ClusterRole` and `ClusterRoleBinding` which allow `flowlogs-pipeline` and `network-observability-plugin` service accounts to read and write the network logs:
+
+    ```yaml
+  ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: lokistack-network-tenant-logs
+    rules:
+    - apiGroups:
+      - 'loki.grafana.com'
+      resources:
+      - network
+      resourceNames:
+      - logs
+      verbs:
+      - 'get'
+      - 'create'
+  ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: lokistack-network-tenant-logs
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: lokistack-network-tenant-logs
+    subjects:
+    - kind: ServiceAccount
+      name: flowlogs-pipeline
+      namespace: network-observability
+    - kind: ServiceAccount
+      name: network-observability-plugin
+      namespace: network-observability
```

Comment:
Do you really use for a namespace with the `openshift-` prefix in the name, i.e. `openshift-network-observability`?